ex13.gen.json

```diff
@@ -0,0 +1,6 @@
+[
+  {"bn":"http://[2001:db8:","bt":1.320078429e+09,"bu":"/","n":":3]/"},
```

Comment:
This is a bit exotic way to split the base name and name. If we want example like this, maybe it should be an example on it's own and discussed in more detail instead of being one of the actuator examples?